apis/content/contentapi.yaml

```diff
@@ -1842,7 +1842,7 @@ paths:
   /content/v1/search:
     post:
       description: |-
-        This API is associated with Searching Object created as part of sourcing object creation on the Sunbird Platform.
+        This API is associated with Searching content and collection objects created as part of sourcing object creation on the Sunbird Platform.
```

Comment:
1. Line 314, 315: Please rectify the formatting of the bullet points. It is appearing as sub-bullets.
2. Global: Please check the punctuation in the line “The Content Type entity is the media type of the resource.Possible media types can be”
3. Global: Please rectify the casing of “To” in the line “Please append a valid Content Id To the Request URL”
4. Line 6: It should be “Sunbird Platform. APIs perform”
5. Line 1848 & 2355: Please rectify the formatting of the line. It is appearing along with the bullet point.
6. Line 1028, 1623: Please rectify the formatting in the line for the success response.
7. Global: Please rectify the punctuations and spacings in all the response descriptions.

Reply:
@anisha147 Not feasible to address punctuations and spacings in all the response descriptions now. We can look into this gradually. However, I have tried to address issues at places similar to the line numbers you have mentioned.